### PR TITLE
feat(useRepairs): enhanced useRepairs hooks with additional tests

### PR DESCRIPTION
**Pull request by RVerma90**

### CODE REVIEW

File: lib/api/repair/v1/service.test.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import { useRepairs } from "./service";
+
+import { RepairsFilters } from "./types";
+
+jest.mock("@mtfh/common/lib/http", () => ({
+  ...jest.requireActual("@mtfh/common/lib/http"),
+  useAxiosSWR: jest.fn(),
+}));
+
+const repairsMock = [
```
Comment (kymmeh) [inline lib/api/repair/v1/service.test.ts:12]: Ideally we want to generate mocks from `mtfh-frontend` which I also need for my PR. Let's catch up on that.

File: lib/api/repair/v1/service.test.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import { config } from "@mtfh/common/lib/config";
+import { useAxiosSWR } from "@mtfh/common/lib/http";
+import { useRepairs } from "./service";
+
+import { RepairsFilters } from "./types";
+
+jest.mock("@mtfh/common/lib/http", () => ({
```
Comment (cloudratha) [inline lib/api/repair/v1/service.test.ts:7]: Why mock the library when we have msw?

File: lib/api/repair/v1/service.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+const repairStatusGroupings: { [key: string]: number[] } = {
+  [RepairsFilters.CANCELLED]: [30],
+  [RepairsFilters.COMPLETED]: [40, 50],
+  [RepairsFilters.IN_PROGRESS]: [20, 60, 80, 90, 100, 110, 120, 1000, 1010, 1080, 1090],
+  [RepairsFilters.LOCKED]: [200],
+  [RepairsFilters.ON_HOLD]: [10, 70],
+};
+
+export const useRepairs = (
```
Comment (cloudratha) [inline lib/api/repair/v1/service.ts:23]: is `useWorkOrders` or `useRepairsWorkOrders` a better name?

File: lib/api/repair/v1/service.test.ts [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    tradeCode: "EL",
+    tradeDescription: "Electrical - EL",
+    status: "In Progress",
+  },
+];
+
+test("useRepairs: it should send the correct query params to the API", async () => {
+  const id = "00075623";
+  (useAxiosSWR as jest.Mock).mockResolvedValueOnce(repairsMock);
+  const response = await useRepairs(id);
```
Comment (cloudratha) [inline lib/api/repair/v1/service.test.ts:32]: Should probably use `@testing-library/react-hooks` for testing hooks.

File: lib/api/repair/v1/service.ts [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  pageNumber = 1,
+  pageSize = 12,
+): AxiosSWRResponse<RepairsResponse> => {
+  const params: RepairsRequestParams = {
+    propertyReference: id,
+    PageNumber: pageNumber,
+    PageSize: pageSize,
+    ...(!!filter && {
+      StatusCode: (repairStatusGroupings as Record<string, number[]>)[`${filter}`].join(
+        "&",
```
Comment (cloudratha) [inline lib/api/repair/v1/service.ts:35]: Wont using ampersand here break the query parameters? shouldn't it be a comma?

File: lib/api/repair/v1/types.ts [hunk 6]
```diff
@@ -0,0 +1,33 @@
+export interface Repair {
```
Comment (cloudratha) [inline lib/api/repair/v1/types.ts:1]: Looking at swagger the api is on v2?

File: lib/api/repair/v1/service.ts (hunk 3 above)
Comment (RVerma90) [reply lib/api/repair/v1/service.ts:23]: Updated to `useWorkOrders`

File: lib/api/repair/v1/types.ts (hunk 6 above)
Comment (RVerma90) [reply lib/api/repair/v1/types.ts:1]: Updated to `v2`

File: lib/api/repair/v1/service.test.ts (hunk 2 above)
Comment (RVerma90) [reply lib/api/repair/v1/service.test.ts]: Removed all together as it's not needed

File: lib/api/repair/v1/service.test.ts (hunk 4 above)
Comment (RVerma90) [reply lib/api/repair/v1/service.test.ts]: Updated to use `@testing-library/react-hooks` for testing hooks

File: lib/api/repair/v1/service.test.ts (hunk 1 above)
Comment (RVerma90) [reply lib/api/repair/v1/service.test.ts]: Updated and using generated mocks instead

File: lib/api/repair/v1/service.ts (hunk 5 above)
Comment (RVerma90) [reply lib/api/repair/v1/service.ts]: As per our chat, the repairs BE currently accept an ampersand for multiple status codes.

**Review by kymmeh — APPROVED** [review]

**Pull request merged**